Apps/W1/Shopify/app/src/Base/Tables/ShpfyLanguage.Table.al

```diff
@@ -0,0 +1,72 @@
+namespace Microsoft.Integration.Shopify;
+
+using System.Globalization;
+
+table 30154 "Shpfy Language"
+{
+    Caption = 'Shopify Language';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Shop Code"; Code[20])
+        {
+            Caption = 'Shop Code';
+            DataClassification = CustomerContent;
+            Editable = false;
+            TableRelation = "Shpfy Shop";
+        }
+
+        field(2; Locale; text[2])
+        {
+            Caption = 'Locale';
+            DataClassification = SystemMetadata;
+            Editable = false;
+        }
+
+        field(3; "Sync Translations"; Boolean)
+        {
+            Caption = 'Sync translations';
+            DataClassification = CustomerContent;
+
+            trigger OnValidate()
+            begin
+                Rec.TestField("Language Code");
+            end;
+        }
+        field(4; "Language Code"; Code[10])
+        {
+            Caption = 'Language Code';
+            TableRelation = Language;
+            DataClassification = CustomerContent;
+
+            trigger OnValidate()
+            begin
```

Comment:
@tinestaric should we verify that the `"Language Code"` assignment is unique?

Reply:
Hmm, valid question. PO's reviewing this item this week, I'll let them decide.